app/views/match_decisions/_match_recommendation_shelter_agency_fields.haml

```diff
@@ -16,6 +16,8 @@
         Client provided a
         = _(@match.client.housing_release_status)
 
+- unless Config.get(:include_note_in_email_default).nil?
```

Comment:
I don't think, you want a required boolean.

Reply:
Maybe I was being lazy, but this is 3-valued logic... if it is set to `true` it defaults to including the note in the email, if it is 'false' it defaults to not being included, but shows the checkbox to allow it to be included. And if it is `nil`, then it it is not included, and the checkbox is hidden.

Reply:
My mistake, I tagged the wrong line.  I meant to tag the input on line 20